app/routes/events/import.js

```diff
@@ -3,5 +3,14 @@ import Route from '@ember/routing/route';
 export default Route.extend({
   titleToken() {
     return this.get('l10n').t('Import');
+  },
+  async model() {
+
+    const data = await this.get('store').findAll('importJob');
```

Comment:
This is not a valid query. There is no importJob model, please try importing the event yourself, and request a review when it works.

Reply:
I have implemented the importJob model both in backend and fronted. Please check fossasia/open-event-server#5514.

Reply:
I have tested it locally, it works.